qiita_db/processing_job.py

```diff
@@ -7,10 +7,62 @@
 # -----------------------------------------------------------------------------
 
 from uuid import UUID
-
+from datetime import datetime
+from subprocess import Popen, PIPE
+from multiprocessing import Process
+from os.path import join
+from itertools import chain
+from collections import defaultdict
+from json import dumps
+
+from future.utils import viewitems, viewvalues
+import networkx as nx
+
+from qiita_core.qiita_settings import qiita_config
 import qiita_db as qdb
 
 
+def _system_call(cmd):
+    """Call cmd and return (stdout, stderr, return_value)
+
+    Parameters
+    ----------
+    cmd : str or iterator of str
```

Comment:
Good catch - when adding the test, I found that it doesn't actually work with an iterable of str. The only reason why we were accepting an iterable is b/c Popen supports both. However, from the [Popen](https://docs.python.org/2/library/subprocess.html#popen-constructor) docs:
> If shell is True, it is recommended to pass args as a string rather than as a sequence
Thus, I think disabling iterable is the best (we just need to update the docs).